src/static/styles/App.scss

```diff
@@ -702,3 +702,108 @@ div.scrollbar {
 	height: 1.25rem;
 	width: 1.25rem;
 }
+
+.client-card {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 1.5rem;
+
+	width: 21.625rem;
+	height: 16.813rem;
+
+	background: #FFFFFF;
+	box-shadow: 0rem 0.25rem 1.25rem rgba(0, 0, 0, 0.1);
+	border-radius: 1.5rem;
+	
+	flex: none;
+	order: 0;
+	flex-grow: 0;
+}
+
+.client-card:hover {
+	background: #FFFFFF;
+	box-shadow: 0rem 0.25rem 1.25rem rgba(0, 0, 0, 0.4);
+	border-radius: 1.5rem;
+}
+
+.right-angle-right {
+	border-bottom-right-radius: 0rem;
+	border-top-right-radius: 0rem;
+}
+
+.right-angle-left {
+	border-bottom-left-radius: 0rem;
+	border-top-left-radius: 0rem;
+}
+
+.pagination-link {
+	color: $primary-theme;
+	float: left;
+	padding: 8px 16px;
+	text-decoration: none;
+	transition: background-color .3s;
+	border: 1px solid #ECEEEF;
+	border-left: none;
+}
+
+.pagination-link-1 {
+	color: $primary-theme;
+	float: left;
+	padding: 8px 16px;
+	text-decoration: none;
+	transition: background-color .3s;
+	border: 1px solid #ECEEEF;
```

Comment:
we should reuse color variables as much as possible- can you make a grey variable if it doesn't already exist